tests/apm/mod.rs

```diff
@@ -121,10 +124,17 @@ fn logging() {
                 _id: 1 }] }\n",
                &line);
 
+
+    let insert_one_line_start = if v3_3 {
+        "COMMAND.insert_one 127.0.0.1:27017 COMPLETED: { ok: 1, n: 1 } ("
+    } else {
+        "COMMAND.insert_one 127.0.0.1:27017 COMPLETED: { n: 1, ok: 1 } ("
```

Comment:
yeah, makes sense as long as we're supposed to print document fields in insert order (most likely) and read server responses in insert order (probably doesn't even mention this in any of the documentation, but we wouldn't want to do any extra work here to alphabetize anyways).